validation/src/main/scala/hmda/validation/engine/lar/LarEngine.scala

```diff
@@ -1,23 +1,27 @@
 package hmda.validation.engine.lar
 
 import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.model.institution.Institution
+import hmda.validation.context.ValidationContext
 import hmda.validation.engine.lar.syntactical.LarSyntacticalEngine
 import hmda.validation.engine.lar.validity.LarValidityEngine
 import hmda.validation.engine.lar.quality.LarQualityEngine
+
 import scalaz._
 import Scalaz._
 
 trait LarEngine extends LarSyntacticalEngine with LarValidityEngine with LarQualityEngine {
 
-  def validateLar(lar: LoanApplicationRegister): LarValidation = {
+  def validateLar(lar: LoanApplicationRegister, ctx: ValidationContext): LarValidation = {
     (
-      checkSyntactical(lar) |@|
-      checkValidity(lar) |@|
-      checkQuality(lar)
+      checkSyntactical(lar, ctx) |@|
+      checkValidity(lar, ctx) |@|
+      checkQuality(lar, ctx)
     )((_, _, _) => lar)
 
   }
 
+  // FIXME: This is not actually used anywhere!  These should really be "macro" edits!
   def validateLars(lars: Iterable[LoanApplicationRegister]): LarsValidation = {
```

Comment:
I was hesitant to remove this as it's not related to this PR.  Shall I create a new issue for this?  It's probably worth a bit of discussion on what it would be replaced with.